ChangeSet 1.748, 2002/10/21 14:37:44-07:00, greg@kroah.com

[PATCH] USB: added support for Clie NX60 device.

Thanks to Hiroyuki ARAKI <hiro@zob.ne.jp> for the information.


diff -Nru a/drivers/usb/serial/visor.c b/drivers/usb/serial/visor.c
--- a/drivers/usb/serial/visor.c	Mon Oct 21 22:14:36 2002
+++ b/drivers/usb/serial/visor.c	Mon Oct 21 22:14:36 2002
@@ -179,6 +179,7 @@
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_4_0_ID) },
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_S360_ID) },
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_4_1_ID) },
+	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_NX60_ID) },
 	{ }					/* Terminating entry */
 };
 
@@ -200,6 +201,7 @@
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_4_0_ID) },
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_S360_ID) },
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_4_1_ID) },
+	{ USB_DEVICE(SONY_VENDOR_ID, SONY_CLIE_NX60_ID) },
 	{ }					/* Terminating entry */
 };
 
diff -Nru a/drivers/usb/serial/visor.h b/drivers/usb/serial/visor.h
--- a/drivers/usb/serial/visor.h	Mon Oct 21 22:14:36 2002
+++ b/drivers/usb/serial/visor.h	Mon Oct 21 22:14:36 2002
@@ -34,6 +34,7 @@
 #define SONY_CLIE_4_0_ID		0x0066
 #define SONY_CLIE_S360_ID		0x0095
 #define SONY_CLIE_4_1_ID		0x009A
+#define SONY_CLIE_NX60_ID		0x00DA
 
 /****************************************************************************
  * Handspring Visor Vendor specific request codes (bRequest values)
